README.md

```diff
@@ -1,9 +1,9 @@
 [![Build Status](https://travis-ci.com/pulumi/pulumi-gcp.svg?token=eHg7Zp5zdDDJfTjY8ejq&branch=master)](https://travis-ci.com/pulumi/pulumi-gcp)
 [![Slack](http://www.pulumi.com/images/docs/badges/slack.svg)](https://slack.pulumi.com)
-[![NPM version](https://badge.fury.io/js/%40pulumi%2Fpulumi.svg)](https://npmjs.com/package/@pulumi/pulumi)
-[![Python version](https://badge.fury.io/py/pulumi.svg)](https://pypi.org/project/pulumi)
-[![GoDoc](https://godoc.org/github.com/pulumi/pulumi?status.svg)](https://godoc.org/github.com/pulumi/pulumi)
-[![License](https://img.shields.io/npm/l/%40pulumi%2Fpulumi.svg)](https://github.com/pulumi/pulumi/blob/master/LICENSE)
+[![NPM version](https://badge.fury.io/js/%40pulumi%2Fpulumi.svg)](https://npmjs.com/package/@pulumi/gcp)
+[![Python version](https://badge.fury.io/py/pulumi.svg)](https://pypi.org/project/pulumi-gcp)
+[![GoDoc](https://godoc.org/github.com/pulumi/pulumi?status.svg)](https://godoc.org/github.com/pulumi/pulumi-gcp)
```

Comment:
Minor thing - but when we are referring to the Go SDKs we typically link to https://godoc.org/github.com/pulumi/pulumi-gcp/sdk/go/gcp. Not sure if that’s the intention here - but I suspect it is?

Reply:
But there are also still links here to Pulumi/Pulumi right (the svg is being generated based on that).